Lecture-2/Tests/calculator/ApplicationTest.java

```diff
@@ -0,0 +1,58 @@
+package calculator;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static task3.Application.getResult;
+
+public class ApplicationTest {
+
+    @Test(dataProvider = "dataForAddition")
+    public void additionTest(double firstDigit, double secondDigit, double result) throws InterruptedException {
+        Thread.sleep(1000);
+        Assert.assertEquals(getResult("+", firstDigit, secondDigit), result, "Addition method failed");
```

Comment:
You should create an instance of calculator to test it. This instance is then used in each unit test
All tests should be independent from each other